polclassifier/interface/main.py

```diff
@@ -136,7 +137,52 @@ def pred_sklearn(speech: str = None) -> np.ndarray:
 
     print(f"✅ And the winner is ... {y_pred}")
 
-    return y_pred
+    # Predict probabilities if the model supports it
+    if hasattr(model, 'predict_proba'):
+        y_prob = model.predict_proba(X_vectorized)
+        print(f"✅ Probability estimates: {y_prob}")
+
+    else:
+        print("❌ Model does not support probability estimates.")
+        y_prob = None
+
+    return y_pred, np.max(y_prob[0])
```

Comment:
```suggestion
    return y_pred, y_prob
```

Reply:
I would reassign 'y_prob' earlier because if it gets defined as None in line 147, then line 149 will return an error